src/modules/catalog.js

```diff
@@ -1869,9 +1880,9 @@ class Catalog {
    * @param {string} parameters.id - Redirect rule ID
    * @param {string} parameters.url - Target URL returned when a match happens
    * @param {object[]} parameters.matches - List of match definitions
-   * @param {string} [parameters.start_time] - Time at which rule begins to apply (ISO8601 format preferred)
-   * @param {string} [parameters.end_time] - Time at which rule stops to apply (ISO8601 format preferred)
-   * @param {string[]} [parameters.user_segments] - List of user segments
+   * @param {string} [parameters.startTime] - Time at which rule begins to apply (ISO8601 format preferred)
+   * @param {string} [parameters.endTime] - Time at which rule stops to apply (ISO8601 format preferred)
+   * @param {object[]} [parameters.userSegments] - List of user segments
```

Comment:
Is this supposed to be a list of objects or a list of strings? Is our [docs](https://docs.constructor.io/rest_api/redirect_rules/#redirect-rule-object) incorrect? 🤔 If so, we should probably create a story to fix it